addons/auth_oauth/static/src/scss/auth_oauth.scss

```diff
@@ -0,0 +1,19 @@
+.o_auth_oauth_providers {
+    font-size: 14px;
+
+    i[class*='facebook'] {
+        color: #3b5998;
+    }
+
+    i[class*='google-plus'] {
```

Comment:
Why not matching the used class directly instead of using an attribute selector? Faster, cleaner, no ugly red on GitHub, ... :) Ok we would have to change the rule if we change the icon class... but that's ok.